docs/xdebug.md

```diff
@@ -37,6 +43,32 @@
 
 Happy debugging!
 
+## Setup for console php
+1. Set environment variable on the `cli` service
+
+    ```yml
+    cli:
+      ...
+      environment:
+        - XDEBUG_ENABLED=1
+        - XDEBUG_CONFIG=idekey=PHPSTORM remote_host=192.168.10.1
+        - PHP_IDE_CONFIG=serverName=drupal7.docksal
+      ...
+    ```
+    
+   You need to replace **drupal7.docksal** with your domain. You can find it in `docksal.env` section:
```

Comment:
You need to replace **drupal7.docksal** with your domain or you can just override it using `docksal.env`: